massa-models/src/node_configuration/default_testing.rs

```diff
@@ -111,18 +111,22 @@ pub const T0: MassaTime = MassaTime::from(32000);
 * };
 */
 
```

Comment:
Maybe here add something like the following consts are normally in config.toml instead of having that comment duplicated

Reply:
As I said, it's already at the top of the file. those comments are more a "Did you looked at the doc? Do you know what it is?" comment.